locations/spiders/digi_telecommunications_my.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+from locations.storefinders.storepoint import StorepointSpider
+
+
+class DiGiTelecommunicationsMYSpider(StorepointSpider):
+    name = "digi_telecommunications_my"
+    item_attributes = {"brand": "DiGi Telecommunications", "brand_wikidata": "Q3268530"}
+    key = "1624a4f7eafca0"
+
+    def parse_item(self, item, location: {}, **kwargs):
+        if item["phone"] is not None:
+            item["phone"] = "+60-" + re.sub(r"^601", "1-", re.sub(r"^03-", "3-", item["phone"]))
```

Comment:
You shouldn't need to do any phone parsing/manipulation. We normalize the phone number during the scraping process.

Reply:
Thanks, I've removed it. I think at one point I was getting errors about some invalid phone numbers and therefore tried to rewrite the numbers manually. I must have confused spiders as it isn't needed for DiGi.